test/functionalTest/cases/0000_ngsi9_subscriptions/notify_format_in_ngsi9_subscriptions.test

```diff
@@ -685,7 +685,7 @@ Content-Type: application/json; charset=utf-8
             "isDomain" : "false"
```

Comment:
Have ou grep the code searching for other 99** cases appart from 9997? E.g. LISTENER2_PORT is associated to 9979.

This should be the list of ports to verify (sorry, probably I should have mentioned this in the issue before...):

```
export CB_PORT=${CB_PORT:-9999}
export COAP_PORT=${COAP_PORT:-5683}
export CP1_PORT=${CP1_PORT:-9801}
export CP2_PORT=${CP2_PORT:-9802}
export CP3_PORT=${CP3_PORT:-9803}
export CP4_PORT=${CP4_PORT:-9804}
export CP5_PORT=${CP5_PORT:-9805}
export LISTENER_PORT=${LISTENER_PORT:-9997}
export LISTENER2_PORT=${LISTENER2_PORT:-9977}
```